### DISCUSSION TITLE
Serve existing directory with s3

### DISCUSSION THREAD

**Discussion by eliphatfs** [seaweedfs/seaweedfs#5798] [General]
I want to serve an existing directory that does not start with `/buckets` with S3 api. I think by modifying the bucket path handling this can be implemented, but after I changed the hard-coded '/buckets' path to '/' in s3 server, it can find the directories inside as a bucket but there is nothing inside the bucket. I wonder what else I need to change, or are there any implications to serve existing directory as s3.

**Comment by chrislusf** [comment]
It was not attempted before and there would be many blockers.
